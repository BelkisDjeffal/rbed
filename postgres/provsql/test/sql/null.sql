\set ECHO none
SET search_path TO provsql_test,provsql;

CREATE TABLE t(x INT);
INSERT INTO t VALUES(NULL);
SELECT add_provenance('t');
CREATE TABLE t2 AS SELECT * FROM t;
SELECT remove_provenance('t2');
SELECT * FROM t2;
DROP TABLE t;
DROP TABLE t2;
