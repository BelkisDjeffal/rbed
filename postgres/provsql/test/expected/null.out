\set ECHO none
 add_provenance 
----------------
 
(1 row)

 remove_provenance 
-------------------
 
(1 row)

 x 
---
  
(1 row)

